.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+<!--
+Thanks for contributing to _fastlane_! Before you submit your pull request, note that the files inside the docs folder are auto-generated.
```

Comment:
are all of them auto generated?

Reply:
I'm not sure, I supposed all of them. @KrauseFx should know better, so I can change the message if needed.

Reply:
The main things we're worried about are the docs covering `actions`, which are all autogenerated, along with the `available-plugins.md` doc.

Reply:
So "... note that the docs covering `actions` and `available-plugins.md` from the `docs` folder are auto-generated." would be a better fit?

Reply:
Or better yet, add a comment to the top of every auto-generated file stating this, like `Fastlane` is actually adding to the `README` file:

> This README.md is auto-generated and will be re-generated every time [fastlane](https://fastlane.tools) is run.

Reply:
> ... note that files in the `docs` folder covering Actions (`actions.md` and `actions/*`) and Plugins (`available-plugins.md`) are auto-generated. Those should be edited at their sources (https://github.com/fastlane/fastlane/ or plugin repository) and will be updated here regularly

Reply:
Love it.

Reply:
Could someone create a PR for the automatically added warning comment from my last message? Or at least point me to the place that generates them?

Reply:
(I would guess this happens in the `Fastfile` of the main repo)